File: JavaScript/JavaScriptFundamentals/04_OperatorsAndExpressions/PointCircleRectangle.js
function solve(args) {
    var x = +args[0];
    var y = +args[1];

    var xCircle = x - 1;
    var yCircle = y - 1;
    var print;
    if ((xCircle * xCircle) + (yCircle * yCircle) <= Math.pow(1.5, 2))
    {
        print = "inside circle ";
        if (x >= -1 && x <= 5 && y <= 1 && y >= -1)
        {
            print += "inside rectangle";
            console.log(print);
        }
        else
        {
            print += "outside rectangle";
            console.log(print);
        }
    }
    else
    {
        print = "outside circle ";
        if (x >= -1 && x <= 5 && y <= 1 && y >= -1)
        {
            print += "inside rectangle";
            console.log(print);
        }
        else
        {
            print += "outside rectangle";
            console.log(print);
        }
    }
}

solve([2.5,
2]);

File: CSharpFundamentals/03_OperatorsАndExpressions/10_PointCircleRectangle/PointCircleRectangle.cs
using System;

class PointCircleRectangle
{
    static void Main()
    {
        double x = double.Parse(Console.ReadLine());
        double y = double.Parse(Console.ReadLine());
        double xCircle = x - 1;
        double yCircle = y - 1;
        
        if ((xCircle * xCircle) + (yCircle * yCircle) <= Math.Pow(1.5, 2))
        {
            Console.Write("inside circle ");
            if (x >= -1 && x <= 5 && y <= 1 && y >= -1)
            {
                Console.Write("inside rectangle");
            }
            else
            {
                Console.Write("outside rectangle");
            }
        }
        else
        {
            Console.Write("outside circle ");
            if (x >= -1 && x <= 5 && y <= 1 && y >= -1)
            {
                Console.Write("inside rectangle");
            }
            else
            {
                Console.Write("outside rectangle");
            }
        }
    }
}

